packages/react-form-state/src/components/List.tsx

```diff
@@ -45,7 +45,8 @@ export default class List<Fields> extends React.Component<
       const innerFields: FieldDescriptors<Fields> = mapObject(
         fieldValues,
         (value, fieldPath) => {
-          const initialFieldValue = initialValue[index][fieldPath];
+          const initialFieldValue = initialValue[index] && initialValue[index][fieldPath] ?
```

Comment:
hmm... do we need the ternary here? Does `initialValue[index] && initialValue[index][fieldPath]` not work on its own?

Reply:
On its own it would return a boolean rather than the value (if the value exists)

Reply:
I'm not sure this is relevant, but the original implementation of `_.get` works with a path, where our implementation does not:
```
> a = { b: { c: 'foo'}}
> _.get(a, 'b.c')
"foo"
> a['b.c']
undefined
```

Reply:
I don't think it will be relevant in our particular case. The main reaosn we were using `get`, AFAICT, is related to undefined behaviour. In terms of the `&&` operator, it will actually return the value without the ternary.

Reply:
👍  today I learned that && don't always return a bool in JS!